lib/recurly/adjustment.ex

```diff
@@ -48,6 +48,30 @@ defmodule Recurly.Adjustment do
   end
 
   @doc """
+  Lists the adjustments from a given account.
+
+  ## Parameters
+
+  - `account_code` String account_code
+
```

Comment:
Could we add options here too and link to the adjustment list docs: https://dev.recurly.com/docs/list-an-accounts-adjustments like it's done here: https://github.com/bhelx/recurly-client-elixir/blob/master/lib/recurly/subscription.ex#L54. Also we changed from using `list` to `stream`. See: https://github.com/bhelx/recurly-client-elixir/blob/master/lib/recurly/subscription.ex#L49 Let's change this to a `stream` function. I want encourage use of the stream interface and if someone wants to do pagination they can use `Recurly.Resource.list`which now returns pages.